config/prod/tom-cvat-alb.yaml

```diff
@@ -0,0 +1,24 @@
+# Provision a load balancer for minidream.synapse.org
+template_path: remote/managed-alb-https.yaml
+stack_name: tom-cvat-alb
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "dream"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+
+   # load balancer configs
+  Ec2InstanceId: "i-04c318988a159582a"
+  SSLCertificateIdArn: "arn:aws:acm:us-east-1:055273631518:certificate/992c2d49-579a-465e-beff-1a08c284f7db"
+  VpcId: "vpc-0e9b80dc470a797d5"   # computevpc
```

Comment:
vpc id seems correct, however the vpc name is  "sandcastle" not "computevpc". please update comment

Reply:
I dont know what it is.  Please help me figure this out

Reply:
i just request you to update the comment to "# sandcastlevpc"